From c62459ea47040d613b7abe58073ef54d9af5fd2e Mon Sep 17 00:00:00 2001
From: Johannes Altmanninger <aclopte@gmail.com>
Date: Sun, 29 Aug 2021 20:10:25 +0200
Subject: [PATCH 4/5] UPSTREAM: Fix detection of zombies in test

Due to the second column the old glob no longer worked.

(cherry picked from commit bd79e753ffa213c7a39b5b66d7b3709ba9837582)
Signed-off-by: Nathan Chancellor <nathan@kernel.org>
---
 tests/checks/jobs.fish | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/checks/jobs.fish b/tests/checks/jobs.fish
index 08fbb2223..766ea7e24 100644
--- a/tests/checks/jobs.fish
+++ b/tests/checks/jobs.fish
@@ -19,7 +19,7 @@ if not contains (uname) SunOS
     ps -o ppid,stat
 else
     ps -o ppid,s
-end | string match -e $fish_pid | string match 'Z*'
+end | string match -e $fish_pid | string match '*Z*'
 
 # Verify disown can be used with last_pid, even if it is separate from the pgroup.
 # This should silently succeed.
-- 
2.34.1

